### PR TITLE
Added comb sort in python

### PR DESCRIPTION
**Pull request by mehabhalodiya**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

- Comb sort is a relatively simple sorting algorithm.
- Comb sort improves on bubble sort algorithm.
- In bubble sort, distance (or gap) between two compared elements is always one.
- Comb sort improvement is that gap can be much more than 1, in order to prevent
slowing down by small values at the end of a list.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

File: Python/sort/comb_sort.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+"""
+CombSort Algorithm Implementation in Python 3
```
Comment (ankitaggarwal23) [inline Python/sort/comb_sort.py]: Rewrite it to `CombSort Algorithm`

File: Python/sort/comb_sort.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+Comb sort improves on bubble sort algorithm.
+In bubble sort, distance (or gap) between two compared elements is always one.
+Comb sort improvement is that gap can be much more than 1, in order to prevent
+slowing down by small values at the end of a list.
+
+Time Complexity :
+Worst case complexity is O(n^2)
+Best Case complexity is O(n log n)
+
+Space Complexity: O(1)
```
Comment (ankitaggarwal23) [inline Python/sort/comb_sort.py]: - Add Time and Space Complexity at the end of the Code

File: Python/sort/comb_sort.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Argument/ Return Type:
+arr : Unorded list
+output : Return list in ascending order
+
```
Comment (ankitaggarwal23) [inline Python/sort/comb_sort.py]: Remove the redundant comments

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
Add a readme update and resolve the merge conflicts as well

**Comment by mehabhalodiya** [comment]
> Add a readme update and resolve the merge conflicts as well

Solved!

File: Python/sort/comb_sort.py (hunk 1 above)
Comment (mehabhalodiya) [reply Python/sort/comb_sort.py]: Yes, changed.

File: Python/sort/comb_sort.py (hunk 2 above)
Comment (mehabhalodiya) [reply Python/sort/comb_sort.py]: Added time and space complexity after sample input/output.

File: Python/sort/comb_sort.py (hunk 3 above)
Comment (mehabhalodiya) [reply Python/sort/comb_sort.py]: Yes, removed those lines!

**Comment by mehabhalodiya** [comment]
@ankitaggarwal23 Kindly review this.
I have done suggested changes.

**Comment by mehabhalodiya** [comment]
@iamrajiv @HarshCasper Kindly review this.

**Review by ankitaggarwal23 — COMMENTED** [review]
Code again plagiarized from [here](https://www.geeksforgeeks.org/comb-sort/).

**Comment by mehabhalodiya** [comment]
No, that's why I have asked you to suggest me something.
Now you only tell me that when will you believe that it's as per my understanding. 

Earlier I have created one function  for logic of sorting algorithm and then I have divide it into two and then calling the above one
So kindly requesting you to check properly.

If you want me to explain you whole piece of code then tell me I will tell you.
This is my sincere request. I am politely asking to you. Please!
Kindly requesting you for the same.
@ankitaggarwal23

**Comment by siddharth25pandey** [comment]
> No, that's why I have asked you to suggest me something.
> Now you only tell me that when will you believe that it's as per my understanding.
> 
> Earlier I have created one function for logic of sorting algorithm and then I have divide it into two and then calling the above one
> So kindly requesting you to check properly.
> 
> If you want me to explain you whole piece of code then tell me I will tell you.
> This is my sincere request. I am politely asking to you. Please!
> Kindly requesting you for the same.
> @ankitaggarwal23

@mehabhalodiya Your code is found exactly copied from the given reference. We can't move any conversation ahead.

**Pull request closed without merging**